res/TemplateBatchFiles/libv3/models/protein/empirical.bf

```diff
@@ -496,6 +506,180 @@ function models.protein.mtMAMML.ModelDescription(type) {
 
 
 
+/**************************************** cpREV functions *************************************/
+
+/**
+ * @name models.protein.cpREV.ModelDescription
+ * @description Create the baseline schema (dictionary) for the cpREV model of protein evolution
+ * @returns {Dictionary} model description
+ * @param {String} type
+ */
+function models.protein.cpREV.ModelDescription(type) {
```

Comment:
Why do these model descriptions take `type` as a parameter? Shouldn't that be hardcoded and not exposed to downstream developers?